argoverse/data_loading/simple_track_dataloader.py

```diff
@@ -54,16 +54,16 @@ def get_log_calibration_data(self, log_id: str) -> Mapping[str, Any]:
         assert isinstance(log_calib_data, dict)
         return log_calib_data
 
-    def get_city_to_egovehicle_se3(self, log_id: str, lidar_timestamp: int) -> Optional[SE3]:
+    def get_city_to_egovehicle_se3(self, log_id: str, timestamp: int) -> Optional[SE3]:
```

Comment:
we can get city_SE3_egovehicle at camera or LiDAR observation time, so this should be agnostic. I made it more generic in the name.